<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cSnmpDevQuery</class>
 <widget class="QWidget" name="cSnmpDevQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelZone">
         <property name="text">
          <string>Zóna :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxZone"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPatPlace">
         <property name="text">
          <string>Szűrés hely névre (SQL minta):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditPatPlace"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelPlace">
         <property name="text">
          <string>Hely :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBoxPlace"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelPatName">
         <property name="text">
          <string>Szűrés a névre (SQL minta):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditPatName"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelNode">
         <property name="text">
          <string>Modosítandó eszköz kiválasztása :</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Eszköz neve (új vagy modosított) :</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelIP">
         <property name="text">
          <string>IP cím :</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelCom">
         <property name="text">
          <string>Community :</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="lineEditCom">
         <property name="text">
          <string>public</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="labelNote">
         <property name="text">
          <string>Megjegzés (az eszközhöz) :</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="lineEditNote"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="labelSnmpV">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>SNMP verzió :</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButtonSnmpV2c">
           <property name="text">
            <string>2c</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSnmpV1">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,0">
         <item>
          <widget class="QComboBox" name="comboBoxNode"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ID :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditNodeId">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="comboBoxIp">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="checkBoxToSnmp">
         <property name="text">
          <string>Konvertálás SNMP eszközzé</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelToSnmp">
         <property name="text">
          <string>Nem SNMP eszközök listázása :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayoutType">
       <item row="0" column="0">
        <widget class="QLabel" name="labelType">
         <property name="text">
          <string>Típus :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonQuery">
       <property name="text">
        <string>SNMP lekérdezés indítása</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Mentés</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
